pkg/util/oc/oc_struct.go

```diff
@@ -127,6 +127,14 @@ func (o OC) CreateTLSSecretWithCACert(t test.TestHelper, ns, name string, keyFil
 	})
 }
 
+func (o OC) CreateGenericSecretFromFile(t test.TestHelper, ns, name string, file string) {
+	t.T().Helper()
+	o.withKubeconfig(t, func() {
+		t.T().Helper()
+		o.CreateGenericSecretFromFiles(t, ns, name, file)
+	})
```

Comment:
No need to wrap in `withKubeconfig`, because the `CreateGenericSecretFromFiles` already wraps it.
```suggestion
	o.CreateGenericSecretFromFiles(t, ns, name, file)
```